server/controllers/lecture-controller.js

```diff
@@ -7,17 +7,19 @@ exports.getBookingLectures = async (req, res) => {
   const studentId = req.user && req.user.id;
   const today = moment().format("YYYY-MM-DD HH:mm:ss");
   const deadline = moment(today).add(12, "hours").format("YYYY-MM-DD HH:mm:ss");
-  //const dateShown = moment(today).add(2, "weeks").format("YYYY-MM-DD HH:mm:ss");
   knex
     .select(
       { id: "lecture.id" },
-      { name: "lecture.name" },
+      //{ name: "lecture.name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.